src/main/kotlin/com/vk/tlschema/highlighting/TLSchemaBreadcrumbsProvider.kt

```diff
@@ -0,0 +1,39 @@
+package com.vk.tlschema.highlighting
+
+import com.intellij.lang.Language
+import com.intellij.openapi.util.NlsSafe
+import com.intellij.psi.PsiElement
+import com.intellij.ui.breadcrumbs.BreadcrumbsProvider
+import com.vk.tlschema.TLSchemaLanguage
+import com.vk.tlschema.psi.TLSchemaConstructorDeclarations
+import com.vk.tlschema.psi.TLSchemaDeclaration
+import com.vk.tlschema.psi.TLSchemaFunDeclarations
+import java.util.*
+
+class TLSchemaBreadcrumbsProvider : BreadcrumbsProvider {
```

Comment:
let's make an `expression body` for all functions.

Reply:
idk I don't like too complex expressions in expression body functions. Such branchy when statements is too complex for me.

Reply:
Okay, but for me:
```kotlin
override fun acceptElement(psiElement: PsiElement) = when (psiElement) {
    is TLSchemaFunDeclarations, is TLSchemaConstructorDeclarations, is TLSchemaDeclaration -> true
    else -> false
}
```

reads better than:
```kotlin
override fun acceptElement(psiElement: PsiElement): Boolean {
    return when (psiElement) {
        is TLSchemaFunDeclarations,
        is TLSchemaConstructorDeclarations,
        is TLSchemaDeclaration -> true

        else -> false
    }
}
```